### PR TITLE
Moment's dead code elimination

### PR DESCRIPTION
**Pull request by plesiecki**
https://github.com/allegro/turnilo/issues/845

### CODE REVIEW

**Review by adrianmroz-allegro — APPROVED** [review]
Possible that we would need to revert it for #504 but I don't think we even know how we will format time. Maybe `Intl` will be common till then.

File: src/replacement/moment-timezone/latest.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (adrianmroz-allegro) [inline src/replacement/moment-timezone/latest.json:1]: Timezones are configurable by user. We shouldn't limit available timezones to [DEFAULT_TIMEZONES](https://github.com/allegro/turnilo/blob/master/src/common/models/customization/customization.ts#L28)

**Review by mkuthan — COMMENTED** [review]
I'm not sure if it is a safe change. See https://allegro.github.io/turnilo/configuration-customizations.html#timezones

File: src/replacement/moment-timezone/latest.json (hunk 1 above)
Comment (plesiecki) [reply src/replacement/moment-timezone/latest.json:1]: I see. It could be user configuration-aware I suppose 🤔

**Review by kzadurska — APPROVED** [review]

**Review by adrianmroz-allegro — APPROVED** [review]

File: src/replacement/moment-timezone/latest.json (hunk 1 above)
Comment (plesiecki) [reply src/replacement/moment-timezone/latest.json]: I've removed this change.

**Pull request merged**